.github/workflows/ci.bazelrc

```diff
@@ -2,6 +2,4 @@
 # Included with a --bazelrc option in the call to bazel in main.yml
 build --announce_rc
 test --test_output=errors
-build --disk_cache=.cache/bazel
```

Comment:
I still don't quite understand why it doesn't work to have this relative to the working directory?

Also, this is one of many spots we need to change, if it's actually wrong. https://github.com/bazel-contrib/rules-template/blob/main/.github/workflows/ci.bazelrc#L10-L11 has been stamped out into a number of rulesets including some of our own.

Reply:
I don't understand either, but it didn't restore the cache.

For environment variables in bazelrc:
https://github.com/bazelbuild/bazel/issues/10904

Reply:
So do we have the same problem in aspect-build/bazel-lib for example? I just want to make sure we don't accidentally fix only a single instance of a general problem.

Reply:
I think they have the same problem we used to have in this repo, which is a directory named $HOME inside the workspace.

![image](https://user-images.githubusercontent.com/3149049/148477924-79f24a5b-ad79-4a05-abc3-e32b9a29b60c.png)

Reply:
And if you look at their caches, they don't change, it's always a cache hit. On the `aspect-build/bazel-lib` we are fortunate that it's a small repo.